honeybee_vtk/writer.py

```diff
@@ -0,0 +1,141 @@
+"""Write vtk files to the disk."""
+
+import os
+import json
+import warnings
+
+from zipfile import ZipFile
+from honeybee.typing import clean_string
+from .hbjson import read_hbjson
+from .hbjson import check_grid, get_grid_base, get_grid_mesh, get_grid_points
+from .tovtk import group_by_face_type
+from .tovtk import write_polydata, write_color_grouped_points, write_arrows
+from .helper import get_face_center
+
+
+def write_vtk(
+        file_path, *, file_name=None, get_grids=True,
+        get_vectors=True):
+
+    """Reads a valid HBJSON and writes a .zip of vtk files to the disk.
```

Comment:
See 4th item here: https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

```suggestion
    """Read a HBJSON and write a .zip of vtk files.
```